README.md

```diff
@@ -144,7 +144,17 @@ To include additional content by glob, define `AdditionalFilesForPackagingFromHe
 </ItemGroup>
 ```
 
-Remapping the content to different output paths is currently not supported.
+For advanced scenarios, such as when the source and target directories don't match exactly, specify both a `SourcePath` and `TargetPath`. `TargetPath` can refer to content metadata using the `^(Metadata)` syntax and can also refer to any metadata from the relative module using `^(HelixModule.^(Metadata)`:
```

Comment:
Did you mean `^(HelixModule).^(Metadata)`?

Reply:
Actually it should be `^(HelixModule.Metadata)`, good catch! Fixed.